config.go

```diff
@@ -62,6 +63,7 @@ type Config struct {
 	Table        string   `toml:"table"`
 	LogLevel     string   `toml:"log-level"`
 	Watch        bool     `toml:"watch"`
+	KubeConfig   string   `toml:"kubeconfig"`
```

Comment:
Do we also need to support handling API endpoint/key/cert files being passed in explicitly or will we handle that in the calico/node by writing out a "local" kubeconfig file?